temporalcli/commands.taskqueue.go

```diff
@@ -10,31 +10,243 @@ import (
 	"go.temporal.io/api/enums/v1"
 	"go.temporal.io/api/taskqueue/v1"
 	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/sdk/client"
 	"go.temporal.io/server/common/tqid"
 )
 
+const taskQueueUnversioned = "UNVERSIONED"
+
+type reachabilityRowType struct {
```

Comment:
```suggestion
type taskQueueReachabilityRowType struct {
```

Since this package is shared by all, might as well qualify the task queue stuff, same with all the little functions, but not a big deal

Reply:
Yes, keep forgetting is single package...